MPU/include/pedals.h

```diff
@@ -32,7 +33,8 @@
 #define MAX_BRAKE_ERRORS            5
 #define MAX_ACCEL_ERRORS            5
 
-#define ACCELERATOR_ERROR_PER       0.10
+#define ACCELERATOR_ERROR_PERCENT   (float)0.1
```

Comment:
Why does this need to be cast to float? What does it define to by default? Is there not a cleaner way to do this?